pandas/io/clipboards.py

```diff
@@ -58,9 +58,14 @@ def read_clipboard(sep=r"\s+", **kwargs):  # pragma: no cover
     # 0  1  2
     # 1  3  4
 
-    counts = {x.lstrip().count("\t") for x in lines}
+    counts = {x.lstrip(" ").count("\t") for x in lines}
     if len(lines) > 1 and len(counts) == 1 and counts.pop() != 0:
         sep = "\t"
+        # check the number of leading tabs in the first line
+        # to account for index columns
+        index_length = len(lines[0]) - len(lines[0].lstrip(" \t"))
+        if index_length != 0:
+            kwargs.setdefault("index_col", [0, index_length - 1])
```

Comment:
I think this won't be correct for a MultiIndex with 3 or more levels. In such a case, don't we want list(range(index_length))?